internal/pkg/service/buffer/api/service/service.go

```diff
@@ -399,6 +406,8 @@ func (*service) Import(d dependencies.ForPublicRequest, payload *buffer.ImportPa
 		// nolint:godox
 		// TODO get sliceID
 
+		// TODO: s.stats.Notify
```

Comment:
Tohle ještě není hotové, jen potřebuju poradit, kdy přesně mám zavolat `Notify`? Pro každý nový record a nebo až když všechny records jsou created? @michaljurecko

Reply:
Máš to zavolať, keď je CSV row bezpečne uložený v etcd, tj. za `CreateRecord`.